src/client/app/redux/slices/appStateSlice.ts

```diff
@@ -104,6 +105,7 @@ export const appStateSlice = createThunkSlice({
 				// Request meter/group/details post-auth
 				dispatch(metersApi.endpoints.getMeters.initiate());
 				dispatch(groupsApi.endpoints.getGroups.initiate());
+				dispatch(weatherLocationApi.endpoints.getWeatherLocationDetails.initiate());
```

Comment:
I'm wondering about getting the weather data on startup. The other ones (meters, groups, conversions, ...) are needed by everyone to use OED. The weather locations are only needed if you are working on them as an admin (and for meters/groups in the future). Normally a useAppSelectoris used in each component where they are needed. Please let me know if I'm missing something here.